interchain-ui/src/vite-env.d.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="vite/client" />
+
+declare module '@agoric/ui-components' {
+  export const stringifyAmountValue;
```

Comment:
Yes I can import `stringifyAmountValue` from `@agoric/web-components` but I had to do a little bit of type wrangling to make it work.

See https://github.com/Agoric/dapp-agoric-basics/pull/18/commits/291515f133d9bb5d24610c6d0527edcc18d76802

Reply:
Ah okay, this is because now `stringifyAmountValue` has a proper type so it's being more strict.

Instead of that wrangling we can improve the types in ui-kit: https://github.com/Agoric/ui-kit/pull/93

Then remove the custom `Purse` type and casting in the `usePurse` hook in this repo, to just rely on the type from ui-kit